src/languages/en.ts

```diff
@@ -2097,17 +2096,18 @@ export default {
         maxAge: ({maxAge}: ViolationsMaxAgeParams) => `Date older than ${maxAge} days`,
         missingCategory: 'Missing category',
         missingComment: 'Description required for selected category',
-        missingTag: ({tagName}: ViolationsMissingTagParams) => `Missing ${tagName ?? 'tag'}`,
+        missingTag: ({tagName}: ViolationsMissingTagParams = {}) => `Missing ${tagName ?? 'tag'}`,
```

Comment:
Something is wrong here

Reply:
Can you elaborate on this? Are you experiencing an error?

Reply:
Uhh, I thought for a moment that we're converting an object to a string, although this object is being destructured. Sorry.

Reply:
Ah ok! Phew. 😅